storybook/storybook-react/src/Screen/Screen.stories.tsx

```diff
@@ -0,0 +1,203 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Blockquote } from '@amsterdam/design-system-react'
```

Comment:
I'm not sure if building an entire page is a good explanation of what `Screen` does. It's a very simple component, I'd keep the stories simple as well

Reply:
Yeah I did it mainly to experiment with the API myself. We can remove it before merging, some if its parts landed in the `GridCell` docs today.

We could also move it to a new ‘Templates’ section already.